packages/core/admin/admin/src/pages/Settings/pages/PurchaseSingleSignOn.tsx

```diff
@@ -16,34 +20,72 @@ const PurchaseSingleSignOn = () => {
             id: 'Settings.sso.title',
             defaultMessage: 'Single Sign-On',
           })}
-          subtitle={formatMessage({
-            id: 'Settings.sso.subTitle',
-            defaultMessage: 'Configure the settings for the Single Sign-On feature.',
-          })}
         />
-        <Box paddingLeft={10} paddingRight={10}>
-          <EmptyStateLayout
-            icon={<EmptyPermissions width="16rem" />}
-            content={formatMessage({
-              id: 'Settings.sso.not-available',
-              defaultMessage:
-                'SSO is only available as part of a paid plan. Upgrade to configure additional sign-in & sign-up methods for your administration panel.',
-            })}
-            action={
-              <LinkButton
-                variant="default"
-                endIcon={<ExternalLink />}
-                href="https://strapi.io/features/single-sign-on-sso?utm_campaign=In-Product-CTA&utm_source=Single-sign-on"
-                isExternal
-                target="_blank"
-              >
-                {formatMessage({
-                  id: 'global.learn-more',
-                  defaultMessage: 'Learn more',
-                })}
-              </LinkButton>
-            }
-          />
+        <Box marginLeft={10} marginRight={10} shadow="filterShadow" hasRadius background="neutral0">
+          <Grid.Root>
+            <Grid.Item col={6} s={12}>
+              <Flex direction="column" alignItems="flex-start" padding={7} gap={2}>
+                <Flex>
+                  <Lock fill="primary600" width={`24px`} height={`24px`} />
+                </Flex>
+                <Flex paddingTop={2} paddingBottom={4}>
+                  <Typography variant="beta" fontWeight="bold">
+                    Simplify authentication for your team
```

Comment:
Shouldn't these labels be translated? Same for the other files

Reply:
You are right! I added them